packages/loader/driver-definitions/src/driverError.ts

```diff
@@ -62,6 +62,13 @@ export enum DriverErrorType {
      * Unexpected response from server. Either JSON is malformed, or some required properties are missing
      */
     incorrectServerResponse = "incorrectServerResponse",
+
+    /**
+     * Epoch version mismatch failures.
+     * This error occurs when the file is modified externally. So the version at the client receiving this error
+     * does not match the one at the server.
+     */
+     epochVersionMismatch = "epochVersionMismatch",
```

Comment:
this is a pretty big change, as it promotes epoch from an odsp concept to a runtime concept

Reply:
if we pass error to dispose we likely don't need this promotion. i'm not against epoch moving up, but i don't know how to rationalize with other concepts we already have like term, or durable seq